config/curricula/wall_jump.yaml

```diff
@@ -0,0 +1,15 @@
+BigWallJump:
+  measure: progress
+  thresholds: [0.1, 0.3, 0.5]
+  min_lesson_length: 100
+  signal_smoothing: true
+  parameters:
+    big_wall_min_height: [0.0, 4.0, 6.0, 8.0]
+    big_wall_max_height: [4.0, 7.0, 8.0, 8.0]
+SmallWallJump:
```

Comment:
Space between this to keep it consistent with `trainer_config.yaml`?

Reply:
Sure, done